alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -0,0 +1,39 @@
+# NodeStatusMaxImagesExceeded
+
+## Meaning
+
+This alert is applicable to clusters that have a large number of container
+images per node. It is triggered when a node exceeds the maximum number of
+configured images (default: 50). Once this limit is reached, the scheduler is
+unable to schedule new pods that require new images on that node.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
+
+## Diagnosis
+
+1. Get the status images count for a given node:
+
+   ```bash
+   $ oc get nodes <node> -o jsonpath='{range .status.images[*]}{.names}{"\n"}{end}' | wc -l
+   ```
+
+   Example output:
```

Comment:
I don't think you need example output for this command.

Reply:
removed